cmd/crank/foundation/util.go

```diff
@@ -0,0 +1,70 @@
+// Package foundation is for functions and helpers shared between components that can be exported to things like crossplane-contrib.
+package foundation
```

Comment:
I am aware of the other discussion regarding the naming of this package, but "foundation" didn't resonate me TBH. How about `common` or `shared` ?

Reply:
+1, i've seen us use `common` in other crossplane projects, that is reasonable to me.

Reply:
and then, is there a better child package this functionality could go in? maybe `common/crd`?